docs/app/Examples/modules/Checkbox/Types/Types.js

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from 'react';
+import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample';
+
+export default class extends Component {
+  render() {
+    return (
+      <div>
+        <ComponentExample
+          title='Checkbox'
+          description='A box for checking.'
+          examplePath='modules/Checkbox/Types/Checkbox'
+        />
+
+        <ComponentExample
+          title='Toggle'
+          description='A checkbox can toggle.'
+          examplePath='modules/Checkbox/Types/Toggle'
+        />
+
+        <ComponentExample
+          title='Slider'
+          description='A checkbox can look like a slider.'
+          examplePath='modules/Checkbox/Types/Slider'
+        />
+
+        <ComponentExample
+          title='Radio'
+          description='A checkbox can be formatted as a radio element. This means it is an exclusive option.'
+          examplePath='modules/Checkbox/Types/Radio'
+        />
+
+        <ComponentExample examplePath='modules/Checkbox/Types/RadioGroup' />
```

Comment:
Notice examples do not have to have titles and descriptions.  By listing multiple ComponentExamples under another with a title/description you can get a nested set of examples:

![image](https://cloud.githubusercontent.com/assets/5067638/10472105/73654f80-71d1-11e5-835d-b669791186d3.png)